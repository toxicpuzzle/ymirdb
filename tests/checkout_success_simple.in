SET a 10
SET b 10
SNAPSHOT
DEL a
LIST ENTRIES
CHECKOUT 1
LIST ENTRIES
bye