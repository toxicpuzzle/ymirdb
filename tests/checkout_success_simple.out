> ok

> ok

> saved as snapshot 1

> ok

> b [10]

> ok

> b [10]
a [10]

> bye
